<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DickerBotHost</class>
 <widget class="QMainWindow" name="DickerBotHost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DickerBotHost</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="device_manager">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Device Manager</string>
    </property>
    <widget class="QComboBox" name="device_manager_port_drop_down">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="device_manager_refresh_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="device_manager_sync_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="device_manager_ssid_data">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="device_manager_password_data">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="device_manager_ssid_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SSID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="device_manager_password_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="socket_manager">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Socket Manager</string>
    </property>
    <widget class="QPushButton" name="socket_manager_start_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="socket_manager_stop_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="socket_manager_status_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="socket_manager_status_data">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>67</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="device_information">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Device Information</string>
    </property>
    <widget class="QLabel" name="device_information_mac_address_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="device_information_mac_address_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="socket_information">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Socket Information</string>
    </property>
    <widget class="QLabel" name="socket_information_ip_address_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="socket_information_port_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="socket_information_ip_address_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="socket_information_port_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
